package.json

```diff
@@ -5,12 +5,14 @@
     "@alephium/cli": "workspace:*",
     "@alephium/web3": "workspace:*",
     "@alephium/web3-test": "workspace:*",
-    "@alephium/web3-wallet": "workspace:*"
+    "@alephium/web3-wallet": "workspace:*",
+    "ws": "^8.5.4"
```

Comment:
The `ws` deps should be in `web3` however can't make it work this way, `js` newbie